test_plugins/validate.py

```diff
@@ -36,8 +38,8 @@ def valid_802_1x(configuration):
                 return False
 
         dot1x_filter = set(['dot1x', 'dot1x_mab', 'dot1x_parking'])
-        for interface in configuration["interfaces"].itervalues():
-            iface_params = set(interface.keys())
```

Comment:
@peterp415 Fixing this for py3 is what started this issue. Attempting to use .items() here leads to .keys() not working, and thus why I used six. I then when wild and thought to just use six everywhere. I can roll back and make a smaller change only using six here

Reply:
I'm not sure what you mean by "not working" there as the views and lists should behave similarly.  But it's really not that important; I'm just being academic about it since I am trying to learn the migration bits myself.

Reply:
welp I had just made a mistake there :sweat: but if this is academic, then wouldn't using six be better? six sends its iteritems to the view based iterator regardless of py2 or py3, which I hear is more efficient. Sure using the vanilla .items() or .values() does work in both py2 and py3, but only py3 uses the view iterator. Also we are using six in the vars plugin, using it here makes things more consistent.